src/templates/globals/banners/facilityBanner/index.tsx

```diff
@@ -0,0 +1,138 @@
+import { useRef, useEffect, useState } from 'react'
+import { recordEvent } from '@/lib/utils/recordEvent'
+import { regionBaseURL } from '@/lib/utils/helpers'
+import { VaBanner } from '@department-of-veterans-affairs/component-library/dist/react-bindings'
+import { NodeMetaInfo } from '@/types/dataTypes/drupal/node'
+import { BannerAlertVacms, Path } from '@/types/index'
+export interface FacilityBannerProps {
```

Comment:
I think we can use `FacilityBannerType` for this purpose and remove this, same for the others.